verifiers/internal/gha/provenance.go

```diff
@@ -297,7 +321,25 @@ func VerifyProvenance(env *dsselib.Envelope, provenanceOpts *options.ProvenanceO
 		if err := isValidDelegatorBuilderID(prov); err != nil {
 			return err
 		}
-		// Note: `provenanceOpts.ExpectedBuilderID` is provided by the user.
+
+		// If ExpectedID is empty, check to see if it is a trusted builder.
+		// If empty and trusted builder, populate with path from provenance,
+		// otherwise, populate from user input.
+		//
+		// Example: the expectedBuilderID will default to the delegator builder ID for BYOB.
+		// This can verify actual BYOB builderIDPath against the trusted builderIDPath inputted,
+		// Currently slsa-framework path is the only one supported for ExpectedBuilderPath.
```

Comment:
nit: I must be getting old and I'm sure I'll die on the hill that says "inputted" is a word.
```suggestion
		// Example: the expectedBuilderID will default to the delegator builder ID for BYOB.
		// This can verify the actual BYOB builderIDPath against the trusted builderIDPath provided.
		// Currently slsa-framework path is the only one supported for ExpectedBuilderPath.
```

Reply:
https://www.merriam-webster.com/grammar/is-inputted-a-word

Seems to be a debated word! Lets go with provided :)